x/gamm/keeper/migrate_test.go

```diff
@@ -1132,13 +1132,13 @@ func (s *KeeperTestSuite) TestCreateCanonicalConcentratedLiquidityPoolAndMigrati
 			desiredDenom0:        USDCIBCDenom,
 			expectError:          types.ErrMustHaveTwoDenoms,
 		},
-		"error: invalid denom durations": {
```

Comment:
note: cannot have this scenerio, lockableDuration must be set at genesis or onChain and we fetch longestDuration afterCreateBalancerPool hook